WcaOnRails/app/models/user.rb

```diff
@@ -155,6 +155,8 @@ def dummy_account?
     wca_id.present? && encrypted_password.blank? && email.casecmp("#{wca_id}@worldcubeassociation.org") == 0
   end
 
+  scope :delegate, -> { where.not(delegate_status: nil) }
```

Comment:
That's what I thought too. User is always singular because it's the model name, but we select a group of users 99% of the time. We just call it @delegates or @users or whatevers.

I'll merge this up for now with .delegates, but we can change in the future if we decide on a convention.